src/mattak/DAQStatus.h

```diff
@@ -77,8 +82,10 @@ namespace mattak
       uint8_t radiant_prescalers_m1[mattak::k::num_radiant_channels] = {0};  //mHz?
       float radiant_scaler_period;
 
-      uint32_t lt_trigger_thresholds[mattak::k::num_lt_channels] = {0}; //
-      uint32_t lt_servo_thresholds[mattak::k::num_lt_channels] = {0}; //
+      uint32_t lt_coinc_trigger_thresholds[mattak::k::num_lt_channels] = {0}; //
+      uint32_t lt_coinc_servo_thresholds[mattak::k::num_lt_channels] = {0}; //
```

Comment:
please increment the version in the ClassDef

Reply:
also, may need manual schema migration if you rename lt_trigger_thresholds to lt_coinc_trigger_thresholds. You can do this in the LinkDef.h, see e.g. https://root.cern/manual/io/#specifying-io-customization-rules-in-a-linkdef-file

Reply:
I guess I could test this with the mattak reader?

#pragma read                                                                        \
    sourceClass="DAQStatus"                                                         \
    source="uint32_t lt_trigger_thresholds[mattak::k::num_lt_channels];             \
        uint32_t lt_servo_thresholds[mattak::k::num_lt_channels]"                   \
    version"[0-4]"                                                                  \
    targetClass="DAQStatus"                                                         \
    target"uint32_t lt_coinc_trigger_thresholds[mattak::k::num_lt_channels];        \
        uint32_t lt_coinc_servo_thresholds[mattak::k::num_lt_channels]"             \
    embed="true"                                                                    \
    include="cstdlib,cstdint"                                                       \
    code="{for(int i=0;i<mattak::k::num_lt_channels;i++){                           \
        lt_coinc_trigger_thresholds[i]=onfile.lt_trigger_thresholds[i];             \
        lt_coinc_servo_thresholds[i]=onfile.lt_servo_thresholds[i];}}"

Reply:
you can test with rno-g-convert on an old data file

Reply:
(ignore previous comment, didn't make sense) 

yes, trying to read an old file with mattak::Dataset is probably easiest (I can do it too, I guess!)

Reply:
note that if any uproot reader reads this, that will need special logic. @fschlueter does the uproot backend attempt to read the thresholds?

Reply:
it does... so if we want to read old files after renaming with uproot we have to check for both cases since I don't think uproot knows about the schema migration logic...

Reply:
oh you haven't committed this to LinkDef.h have you?